demisto_sdk/commands/common/hook_validations/script.py

```diff
@@ -168,3 +170,15 @@ def is_docker_image_valid(self):
         if docker_image_validator.is_docker_image_valid():
             return True
         return False
+
+    def is_valid_name(self):
+        # type: () -> bool
+        if not is_v2_file(self.current_file):
+            return True
+        else:
+            name = self.current_file.get('name')
+            correctName = " v2"
```

Comment:
name of scripts cannot contain whitespace.
@glicht how should v2 automations names end? v2/V2?